app/src/main/java/com/example/isioyemohammed/gitlist/presenter/GithubUsersPresenter.java

```diff
@@ -28,36 +25,48 @@
     /**
      * Context - creates an instance of Context.
      */
-    Context context;
+    ViewUsers view;
 
     /**
      * Class method for provided context to the presenter.
-     * @param context - Instance of Context
+     * @param view - Instance of Context
      */
-    public GithubUsersPresenter(Context context) {
-        this.context = context;
+    public GithubUsersPresenter(ViewUsers view) {
+        this.view = view;
         if (this.githubService == null) {
             this.githubService = new GithubService();
         }
     }
 
+    /**
+     * View interface.
+     */
+    public interface ViewUsers {
+        /**
+         * displayGithubUsers interface method.
+         *
+         * @param developerList - method parameter
+         */
+        void displayGithubUsers(ArrayList<GithubUsers> developerList);
+    }
+
     /**
      * Method for getting list of users from server and populating the recyclerView.
-     * @param recyclerView - recyclerView
      */
-    public void getGitHubUsers(final RecyclerView recyclerView) {
+    public void getGitHubUsers() {
         githubService
                 .getAPI()
                 .getItems()
                 .enqueue(new Callback<GithubUsersResponse>() {
                     @Override
                     public void onResponse(@NonNull Call<GithubUsersResponse> call,
                                            @NonNull Response<GithubUsersResponse> response) {
-                        List<GithubUsers> githubUsersResponse = response.body().getItems();
+                        GithubUsersResponse userResponse = response.body();
+                        ArrayList<GithubUsers> githubUsersResponse;
+                        assert userResponse != null;
```

Comment:
@codestaintin 
Could you please explain the purpose of this line of code? It looks like a JUnit test.

Reply:
It is an alternative to and if-else statement, I chose that approach to reduce the lines of code. Is it a good practice.